src/fortran_routing/diffusive_v02frwk/diffusive_cnt/diffusive_cnt.f90

```diff
@@ -278,11 +278,47 @@ subroutine diffnw(dtini_g, t0_g, tfin_g, saveinterval_ev_g, dt_ql_g, dt_ub_g, dt
                 q_sk_multi=1.0
                 slope = (z(ncomp-1,j)-z(ncomp,j))/dx(ncomp-1,j)
                 if (slope .le. 0.0001) slope = 0.0001
+		oldQ(ncomp,j)= iniq(ncomp,j)
```

Comment:
We seem to be writing data from iniq to oldQ in several different places. Why can't we do this once and get it out of the way? e.g. this happens again at line 293...